src/utils.ts

```diff
@@ -186,7 +181,15 @@ export function getUrl(uri, gateway = gateways[0]) {
 
 export async function getJSON(uri, options: any = {}) {
   const url = getUrl(uri, options.gateways);
-  return fetch(url).then((res) => res.json());
+  return fetch(url, {
+    timeout: 30e3,
+    parseResponse: JSON.parse,
```

Comment:
why do we need this prop only for `getJSON` and `ipfsGet`?

Reply:
Removed it, found another way to handle this